misc/kind/cluster.yaml

```diff
@@ -176,3 +180,15 @@ nodes:
     labels:
       materialize.cloud/disk: true
       materialize.cloud/availability-zone: "3"
+  - role: worker
+    labels:
+      materialize.cloud/disk: true
+      environmentd: true
+  - role: worker
+    labels:
+      materialize.cloud/disk: true
+      environmentd: true
+  - role: worker
+    labels:
+      materialize.cloud/disk: true
```

Comment:
there should be no disk annotation for the supporting-services node.

Reply:
Removed, but if there is any strange error regarding PVC, we should revisit this place 😇